sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedReadSuite.scala

```diff
@@ -59,6 +60,77 @@ class BucketedReadSuite extends QueryTest with SQLTestUtils with TestHiveSinglet
     }
   }
 
+  test("read partitioning bucketed tables with bucket pruning filters") {
```

Comment:
Tried it but it sounds like the bucket ID of the returned rows do not match the bucket ID generated in the test cases. I will continue investigating this issue.